keyboards/redragon/k552/v2/via_json/k552_iso.json

```diff
@@ -0,0 +1,223 @@
+{
```

Comment:
linting fails for this because technically via jsons shouldn't be part of the repo. The build system picks up `keyboards/redragon/k552/v2/rules.mk` and assumes that `keyboards/redragon/k552/v2` is a target, while `keyboards/redragon/k552/v2/iso` is the target. It should probably be better to have a `rules.mk` inside the `iso/ansi` subdirectory instead, assuming you want the via json to live in `v2/via_json` ( although we will move all of the via configs in a seperate repo soon)